schema/asset_uploads/create_asset_request_mutation.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+import { GraphQLString, GraphQLNonNull, GraphQLObjectType } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+
+export const CredentialsType = new GraphQLObjectType({
+  name: "Credentials",
+  description: "An asset which is assigned to a consignment submission",
+  fields: {
+    credentials: {
```

Comment:
aw yea cc @izakp Metaphysics getting the S3 key from the newly added `credentials`

Reply:
These are one-off keys, meant for just 1 upload, right?

Reply:
Yeah, they're size limited too